.azure-ci.yml

```
@@ -0,0 +1,159 @@
+    inputs:
+      restoreSolution: $(Build.SourcesDirectory)\AspNetWebHooks.sln
+      feedsToUse: config
+      nugetConfigPath: $(Build.SourcesDirectory)\.nuget\NuGet.config
+  - script: .\build.cmd
+            EnableSkipStrongNames
+            $(BuildScriptArgs)
+            /bl:artifacts/logs/Build-SkipStrongNames.binlog
+    displayName: Build (Skip StrongNames)
+  - script: .\build.cmd
+            BuildPackages
+            /p:Configuration=CodeAnalysis
+            $(BuildScriptArgs)
+            /bl:artifacts/logs/Build-CodeAnalysis.binlog
+    displayName: Build (CodeAnalysis)
+  - script: .\build.cmd
+            BuildPackages
+            /p:Configuration=Release
+            $(BuildScriptArgs)
+            /bl:artifacts/logs/Build-Release.binlog
+    displayName: Build (Release)
+  - task: CopyFiles@2
+    displayName: Copy packages to artifacts/
+    inputs:
+      sourceFolder: $(Build.SourcesDirectory)/bin/
+      contents: 'Release/**/*.nupkg'
+      targetFolder: $(Build.SourcesDirectory)/artifacts/
+  - task: CmdLine@2
+    inputs:
+      script: 'git rev-parse HEAD > artifacts/commit'
+    displayName: Create commit file
+  - task: PublishBuildArtifacts@1
+    displayName: Upload artifacts
+    condition: eq(variables['system.pullrequest.isfork'], false)
+    continueOnError: true
+    inputs:
+      pathtoPublish: $(Build.SourcesDirectory)/artifacts/
+      artifactName: artifacts-AspnetWebHooks
+      artifactType: Container
+      parallel: true
+
+- ${{ if eq(variables['System.TeamProject'], 'internal') }}:
+  - job: Webhooks_Signed_Build
+    displayName: Build AspnetWebHooks-Signed
+    dependsOn: Webhooks_Build
+    timeoutInMinutes: 60
+    workspace:
+      clean: all
+    pool:
+      name: NetCoreInternal-Pool
+      queue: BuildPool.Server.Amd64.VS2019
+    variables:
+      TeamName: AspNetCore
+      _SignType: real
+      DOTNET_CLI_TELEMETRY_OPTOUT: 1 # Skip signing telemetry to work around an error 
+    steps:
+    - checkout: AspnetWebHooks-Signed
+      clean: true
+    - task: MicroBuildSigningPlugin@2
+      displayName: Install MicroBuild Signing plugin
+      inputs:
+        signType: $(_SignType)
+        zipSources: false
+        feedSource: https://dnceng.pkgs.visualstudio.com/_packaging/MicroBuildToolset/nuget/v3/index.json
+    - task: DownloadBuildArtifacts@0
+      displayName: Download artifacts
+      inputs:
+        artifactName: artifacts-AspnetWebHooks
+        downloadPath: $(Build.StagingDirectory)/downloaded_artifacts/
+    - script: .\build.cmd
+              /t:BuildProjects
+              /p:Sign=Sign
+              /p:SignType=$(_SignType)
+              /p:DropSource=$(Build.StagingDirectory)/downloaded_artifacts/artifacts-AspnetWebHooks/
+              $(BuildScriptArgs)
+              /bl:artifacts/logs/Build-Sign.binlog
+      displayName: Build & Sign
+    - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
```

nit: the public check is not needed because this job runs only in internal builds
```suggestion
    - ${{ if notin(variables['Build.Reason'], 'PullRequest') }}:
```